Source/WebCore/Modules/highlight/Highlight.cpp

```diff
@@ -55,52 +55,54 @@ Ref<Highlight> Highlight::create(FixedVector<std::reference_wrapper<WebCore::Abs
 
 Highlight::Highlight(FixedVector<std::reference_wrapper<WebCore::AbstractRange>>&& initialRanges)
 {
-    m_rangesData.reserveInitialCapacity(initialRanges.size());
+    m_highlightRanges.reserveInitialCapacity(initialRanges.size());
     for (auto& range : initialRanges) {
         repaintRange(range.get());
-        m_rangesData.uncheckedAppend(HighlightRangeData::create(Ref { range.get() }));
+        m_highlightRanges.uncheckedAppend(HighlightRange::create(Ref { range.get() }));
     }
 }
 
 void Highlight::initializeSetLike(DOMSetAdapter& set)
 {
-    for (auto& rangeData : m_rangesData)
-        set.add<IDLInterface<AbstractRange>>(rangeData->range());
+    for (auto& highlightRange : m_highlightRanges)
+        set.add<IDLInterface<AbstractRange>>(highlightRange->range());
 }
 
 bool Highlight::removeFromSetLike(const AbstractRange& range)
 {
-    return m_rangesData.removeFirstMatching([&range](const Ref<HighlightRangeData>& current) {
+    return m_highlightRanges.removeFirstMatching([&range](const Ref<HighlightRange>& current) {
```

Comment:
Optional nit: The template arg for `Ref` probably doesn't need specifying, but that stretches the scope of this PR too far... maybe change that in a forthcoming refactor PR 😛